src/lib/api-utils.js

```diff
@@ -8,11 +8,6 @@ import path from 'path'
 const yaml = require('js-yaml')
 import prisma from './prisma'
 
-const { Video } = new Mux(
```

Comment:
Any reason for this change?

Reply:
That was while I was testing it locally since I don't have Mux credentials. I import a function from this file so when the Video declaration wasn't in the function it was trying to create the instance and log in, which failed and stopped the rest of the script from running. I'll revert that change though since you'll probably have those credentials